.github/actions/coverage-ci/action.yml

```diff
@@ -76,7 +76,13 @@ runs:
         -DF3D_PLUGIN_BUILD_USD=ON
         -DF3D_PLUGIN_BUILD_VDB=ON
         -DF3D_STRICT_BUILD=ON
+        -DF3D_TESTING_ENABLE_EXTERNAL_EGL=ON
+        -DF3D_TESTING_ENABLE_EXTERNAL_GLFW=ON
+        -DF3D_TESTING_ENABLE_EXTERNAL_GLX=ON
+        -DF3D_TESTING_ENABLE_EXTERNAL_OSMESA=ON
+        -DF3D_TESTING_ENABLE_EXTERNAL_QT=OFF
```

Comment:
Should I install QT on the Docker image to test it? @mwestphal

Reply:
Yes, why not  :)